src/lib/Microsoft.Health.Expressions/FromUnixTimestampFunction.cs

```diff
@@ -0,0 +1,33 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System;
+using DevLab.JmesPath.Functions;
+using Newtonsoft.Json.Linq;
+
+namespace Microsoft.Health.Expressions
+{
+    public class FromUnixTimestampFunction : JmesPathFunction
+    {
+        public FromUnixTimestampFunction()
+            : base("fromUnixTimestamp", 1)
```

Comment:
I think it would be safer to add in a new function `fromUnixTimestampMs` in order to handle milliseconds. My concern is being able to understand the unit of the incoming data. I know that you mentioned counting characters, but what if they enter the number `1000`? Do they mean 1000 seconds from the Epoch, or do they mean 1000 milliseconds (i.e. 1 second)?

Reply:
They can also use this method as-is and simply convert their milliseconds into seconds:

Given

```json
{
   "unixtimestampInMs" : "1000"
}
```
They could write:

`fromUnixTimestamp( floor( divide(unixtimestampInMs, `1000` ) ))`

Reply:
I've gone ahead and added a new function to parse unix timestamps in milliseconds